### DISCUSSION TITLE
Does JAX work well on AMD CPUs?

### DISCUSSION THREAD

**Discussion by cool-RR** [jax-ml/jax#23448] [Q&A] [answered]
I'm thinking of buying a new workstation to run my JAX experiments. No GPU, just a high-end CPU. Because of the [recent problems](https://www.techradar.com/computing/cpu/intels-woes-with-core-i9-cpus-crashing-look-worse-than-we-thought-team-blue-really-needs-to-act-now-to-fix-this-mess) with high-end Intel CPUs, I'm considering buying an AMD CPU, like Ryzen 7 7800X3D. Do you think there are any issues with running JAX on AMD CPUs? Maybe some optimizations won't work there?

**Comment by hawkinsp** [comment, marked as answer]
We use AMD CPUs all the time. No problems expected.
